### PR TITLE
Added Ratings form 

### LINKED ISSUE #69: Task.24 - Rating button
Add a rating a button for a student to rate another student in their group on a scale from 1 to 7.

When student A clicks on student B while viewing their group they should have the ability to rate them on a scale of 1-7.

### LINKED ISSUE #70: Task.25 - Alert message (confirmation)
After rating a student, the user should have a confirmation for submitting their rating to confirm that they actually want to submit the rating they have entered. The score should be reiterated as a number out of the maximum of the scale but also as a percentage.

For example, if the scale is from 1 to 4 and the rating is 2 out of 4 then the confirmation should display `Confirm a score of 2/4 (50%)`

### PR DESCRIPTION
**Pull request by Vinuyans**
Added Rating forms with confirmation button and express route

### CODE REVIEW

File: client/src/components/student/RatingModal.jsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  className="text-xl border-solid border-2 w-fit p-2 rounded-md self-center"
+  onClick={() => setIsReviewing(true)}
+>
+  Add Review
+</button>
+<Modal open={isReviewing} onClose={() => setIsReviewing(false)}>
+  <RatingModal
+    handleClose={() => setIsReviewing(false)}
+  />
+</Modal> */}
```
Comment (Marian-Hristov) [inline client/src/components/student/RatingModal.jsx:112]: Is this for when @Kerryki stuff is ready?

File: client/src/index.css [hunk 2]
```diff
@@ -2,53 +2,55 @@
...
 
 .navbar ul li a:hover {
-    color: #00d5ff;
+  color: #00d5ff;
+}
+
+/* Seems to be a bug where the stars are vertical if this is not here */
+svg.star-svg {
+  display: inline;
 }
```
Comment (Marian-Hristov) [inline client/src/index.css:56]: i see

**Review by Marian-Hristov — APPROVED** [review]
Great work as usual, thank you for adding the commented code so that @Kerryki's is easier to merge!

File: client/src/components/student/RatingModal.jsx (hunk 1 above)
Comment (Vinuyans) [reply client/src/components/student/RatingModal.jsx:112]: Yup it needs to be added on a user in a group

**Pull request merged**